util/container/Dockerfile

```diff
@@ -31,6 +31,19 @@ RUN rustup override set 1.77.2
 # Install Bender
 RUN cargo install bender --version 0.28.1
 
+# Install pre-built spike-dasm
+RUN wget https://github.com/pulp-platform/riscv-isa-sim/releases/download/snitch-v0.1.0/snitch-spike-dasm-0.1.0-x86_64-linux-gnu-ubuntu18.04.tar.gz && \
+    tar xzf snitch-spike-dasm-0.1.0-x86_64-linux-gnu-ubuntu18.04.tar.gz && \
+    rm snitch-spike-dasm-0.1.0-x86_64-linux-gnu-ubuntu18.04.tar.gz
```

Comment:
I learned from Ryan that the spike should match the hardware itself so that why we always install spike after building the hardware. Is this the way how spike works?
Have you tried if it works using this pre-build spike with the traces?

Reply:
I would be interested to know how the spike build is affected by the hardware configuration?
Can you show me where?

Reply:
@JosseVanDelm honestly I'm not sure too but I only inferred that it needed it:

https://kuleuven-micas.github.io/snax_cluster/tutorial/other_tools.html

We had these steps because to install `spike`, it needs the `work-vlt` directory which is only generated after a hardware build. That's where they put the spike installation thing. 

Could you check if you could generate traces normally? Like after a software run and the `.dasm` logs are generated, do the `make traces` things and maybe see if you can see the logs. It would be helpful if we don't install it every podman run 😄 so if this `spike` pre-build is fine then yehey!

Reply:
As discussed it seems right now the reason this is implemented this way is because the fesvr part of spike is used to build the simulation platform, but it does not appear like any hardware specific stuff is generated to be consumed by the spike-dasm build process